deployment/README.md

```diff
@@ -1,379 +1,466 @@
+           export PATH=$PATH:/usr/local/go/bin
+         ```
+    - Install [Terraform Engine](https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/tree/master/docs/tfengine#installation), for example:
+         ```shell
+         VERSION=v0.4.0 && \
+           sudo wget -O /usr/local/bin/tfengine https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/releases/download/${VERSION}/tfengine_${VERSION}_linux-amd64 && \
+           sudo chmod +x /usr/local/bin/tfengine
+         ```
+    - Install [Git](https://github.com/git-guides/install-git), for example:
+         ```shell
+         sudo apt-get install git-all
+         ```
+1. [Duplicate](https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/duplicating-a-repository) the [FDA MyStudies repository](https://github.com/GoogleCloudPlatform/fda-mystudies), then clone it locally
+1. Update [`/deployment/deployment.hcl`](/deployment/deployment.hcl) with the values for your deployment
+1. Update [`/deployment/scripts/set_env_var.sh`](/deployment/scripts/set_env_var.sh) for your deployment, then use the script to set your environment variables, for example:
     ```
-
-1. Save the path to your copy of the deployment template and the path to the
-    `mystudies.hcl` solution template to environment variables for later easy
-    referencing.
-
-    ```bash
-    export ENGINE_CONFIG=/path/to/your/local/deployment.hcl
-    export MYSTUDIES_TEMPLATE=/path/to/your/local/mystudies.hcl
+    source set_env_var.sh         # executed from your /deployment/scripts directory
     ```
-
-### Step 1: Deploy Devops project and CICD manually
-
-1. Execute the `tfengine` command to generate the configs. By default, CICD
-    will look for Terraform configs under the `terraform/` directory in the
-    GitHub repo, so set the `--output_path` to point to the `terraform/`
-    directory inside the local root of your GitHub repository.
-
-    Make sure in your first deployment in a new folder, `enable_gcs_backend` in
-    `$MYSTUDIES_TEMPLATE` is set to `false` or commented out.
-
+1. Authenticate as a user with the permissions described above (this deployment assumes gcloud and Terraform commands are made as a user, rather than a service account)
+    - Login and update your [application default credentials](https://cloud.google.com/docs/authentication/production), for example you could run `gcloud auth login --update-adc` (when using a Google Compute Engine VM you must update the application default credentials, otherwise requests will continue to be made with its default service account)
+    - Remember to run `gcloud auth revoke` to log your user account out once your deployment is complete
+
+### Set up your CICD pipelines
+
+1. Generate your Terraform configuration files
+    - Set the `enable_gcs_backend` flag in [`mystudies.hcl`](/deployment/mystudies.hcl) to `false`, for example:
+         ```bash
```

Comment:
I think changing the values manually is probably easier for most people. Using commands is more error prone and may not behave the same way in all environments/OS.